eduNEXT/xblock-mindmap
# feat: add functionality to create static mind maps from Studio

### Description
This PR adds functionality for instructors to create static mind maps from Studio, and students can view them in the LMS.

### How To Test
1. Install this branch in your environment.
2. Add a new block from Studio and edit the field `Is Static`, set it to `True`
3. Create a new mind map in the block, and `Save`.
4. In the LMS you should see the mind map created, but you cannot edit it, nor does the `Save` button appear.

### Examples
#### Studio
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/369c167d-5440-46be-867e-ce11087d4ddd)
#### LMS
![image](https://github.com/eduNEXT/xblock-mindmap/assets/64033729/e928da99-dbad-4b9a-b57c-9dd62b0cd5bf)
